spec/controllers/application_controller/ci_processing_spec.rb

```diff
@@ -90,6 +90,44 @@
     end
   end
 
+  describe "#supports_reconfigure_disks?" do
+    let(:vm) { FactoryGirl.create(:vm_redhat) }
+    let(:rec_ids) { [vm.id] }
+    let(:supports_reconfigure_disks) { true }
+    before(:each) do
+      allow_any_instance_of(ManageIQ::Providers::Redhat::InfraManager::Vm)
```

Comment:
The test was updated to reflect the removal of the instance variable from previous patch which stored the value of the tested method.